### DISCUSSION TITLE
PP-ChatOCRv2是只能提工单让百度来搭建吗？

### DISCUSSION THREAD

**Discussion by zouhan6806504** [PaddlePaddle/PaddleOCR#12686] [Q&A]
如果只想使用里面pdf解析文本的功能，达到 PP-ChatOCRv2类似的结果，能直接使用paddleocr里面的接口吗？

**Comment by a31413510** [comment]
您好，要实现这个功能，现在不支持直接使用paddleocr里面的接口，可以试试在PaddleX中使用这个功能
